src/components/Xmp.react.js

```diff
@@ -23,7 +24,12 @@ const Xmp = (props) => {
 
```

Comment:
It would make sense to also omit `loading_state` from the props being passed down to the child

Reply:
What do you mean?

Reply:
Ah never mind - I'll add it to omit

Reply:
Actually, that doesn't make sense to me - why would we omit the `loading_state`? The Loading component looks at that prop to determine if it's loading, so if it's not there, it won't work.